nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/BetaInc.java

```diff
@@ -0,0 +1,42 @@
+package org.nd4j.linalg.api.ops.custom;
+
+import lombok.NonNull;
+import org.nd4j.autodiff.samediff.SDVariable;
+import org.nd4j.autodiff.samediff.SameDiff;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.api.ops.DynamicCustomOp;
+
+public class BetaInc extends DynamicCustomOp {
+
+    public BetaInc() {
+        super();
+    }
+
+    public BetaInc(@NonNull INDArray a_input, @NonNull INDArray b_input, @NonNull INDArray x_input,
+                   @NonNull INDArray output) {
+        inputArguments.add(a_input);
+        inputArguments.add(b_input);
+        inputArguments.add(x_input);
+        outputArguments.add(output);
```

Comment:
`addInputArgument(a_input, b_input, x_input)` - varargs method
Output may be null